WISE-Community/WISE
# Link to Open Authoring Tool From Classroom Monitor

Steps to Test
1. Login as teacher and launch a run
2. In Classroom Monitor, Click on the Authoring Tool Link. 
3. A Windows Confirm dialogue should warn whether the teacher wants to edit run with students working on it.
4. Upon clicking OK, it should open a new tab showing the authoring tool with correct run ID


Extra Tests
In Classroom monitor, if you click a menu, then click the **Edit Unit Content** a prompt will show up and ask for confirmation to proceed. Accepting to Declining should not break the view of the menus that was selected.
closes #2102
